src/util/fetchUtils.js

```diff
@@ -87,8 +87,8 @@ export const getFetchArgs = (apiUrl, queries, meta) => {
 	const isFormData = queries[0].params instanceof FormData;
 	const isDelete = method === 'DELETE';
 
-	const fetchUrl = new URL(apiUrl);
-	fetchUrl.searchParams.append('queries', rison.encode_array(queries));
+	const searchParams = new URLSearchParams();
```

Comment:
`URL` is used because `URLSearchParams` doesn't have a good browser polyfill. Instead, I recommend doing something like

```js
const fetchUrl = new URL(window.location.origin);
fetchUrl.pathname = `apiUrl`;
```

and leaving the rest of the implementation intact